bindata/network/ovn-kubernetes/ovnkube-node.yaml

```diff
@@ -190,6 +190,11 @@ spec:
             sleep 5
           done
 
+          if [[ ${db_ip} == *":"* ]]; then
+            # IPv6
+            db_ip="[${db_ip}]"
```

Comment:
it appears that the `db_ip` variable isn't actually used any more (other than being checked whether it's the empty string), so we don't need this commit (`Add square brackets around OVN DB IPv6 addresses`) either

Reply:
Thanks, this went through a few rebases.  I'll remove that commit.